### PR TITLE
 docs(color-picker): update mode type

### PR DESCRIPTION
**Pull request by jingjing20**
修复 color-picker 组件文档 Typescript 类型错误

### 🤔 This is a ...

- [ ] 🆕 New feature
- [ ] 🐞 Bug fix
- [x] 📝 Site / documentation improvement
- [ ] 📽️ Demo improvement
- [ ] 💄 Component style improvement
- [ ] 🤖 TypeScript definition improvement
- [ ] 📦 Bundle size optimization
- [ ] ⚡️ Performance optimization
- [ ] ⭐️ Feature enhancement
- [ ] 🌐 Internationalization
- [ ] 🛠 Refactoring
- [ ] 🎨 Code style optimization
- [ ] ✅ Test Case
- [ ] 🔀 Branch merge
- [ ] ⏩ Workflow
- [ ] ❓ Other (about what?)

### CODE REVIEW

**Review by afc163 — DISMISSED** [review]

File: components/color-picker/index.zh-CN.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ group:
 | disabledFormat | 禁用选择颜色格式 | boolean | - |
 | destroyTooltipOnHide | 关闭后是否销毁弹窗 | `boolean` | false | 5.7.0 |
 | format | 颜色格式 | `rgb` \| `hex` \| `hsb` | `hex` | |
-| mode | 选择器模式，用于配置单色与渐变 | `('single' \| 'gradient')[]` | `single` | 5.20.0 |
+| mode | 选择器模式，用于配置单色与渐变 | `'single' \| 'gradient' \| ('single' \| 'gradient')[]` | `single` | 5.20.0 |
```
[the remark this answers is not in the document]
Comment (afc163) [reply components/color-picker/index.zh-CN.md:56]: 英文也改一下？

[the remark this answers is not in the document]
Comment (jingjing20) [reply components/color-picker/index.zh-CN.md:56]: 好的

[the remark this answers is not in the document]
Comment (jingjing20) [reply components/color-picker/index.zh-CN.md:56]: 英文也改好了

[the remark this answers is not in the document]
Comment (thinkasany) [reply components/color-picker/index.zh-CN.md:56]: pr的标题可以改一下  docs(color-picker): update mode type

**Review by afc163 — APPROVED** [review]

**Pull request merged**